centreon-gorgone/tests/robot/config/includer.yaml

```diff
@@ -0,0 +1,3 @@
+name: config.yaml
+description: Configuration of centreon-gorgone. Use config.d directory to change configuration
+configuration: !include /etc/centreon-gorgone/@UNIQIDFROMROBOTTESTINCONFIGFILE@/config.d/*.yaml
```

Comment:
some underscores could make the macro more readable :)
ex: `@UNIQ_ID_FROM_ROBOT_TESTING_CONFIG_FILE@`